Description: This patch fixes the FTBFS by including sys/stat.h.
Origin: vendor
Bug-Debian: http://bugs.debian.org/625005
Forwarded: no
Author: "brian m. carlson" <sandals@crustytoothpaste.net>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-10-05

diff -ur cdrdao.old/dao/ScsiIf-linux.cc cdrdao-1.2.3/dao/ScsiIf-linux.cc
--- cdrdao.old/dao/ScsiIf-linux.cc	2011-05-18 22:00:28.000000000 +0000
+++ cdrdao-1.2.3/dao/ScsiIf-linux.cc	2011-05-18 22:05:38.000000000 +0000
@@ -27,6 +27,7 @@
 #include <assert.h>
 #include <sys/ioctl.h>
 #include <glob.h>
+#include <sys/stat.h>
 #include <asm/param.h>
 #include <scsi/scsi.h>
 #include <scsi/sg.h>
